fixtures/_version.py

```diff
@@ -0,0 +1,16 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
The comment says to not track it, but it's checked in?

Reply:
Thanks, removed it.